compiler/modules/nback.llvm/llvm.bal

```diff
@@ -85,6 +85,24 @@ public function constNull(PointerType ty) returns PointerValue {
     return new PointerValue(ty, "null");
 }
 
+// Corresponds to LLVMContextRef
+public class Context {
+    // Corresponds to LLVMContextCreate
+    public function init() {
+        
+    }
+
+    public function createModuleInContext() returns Module {
+        return new(self);
+    }
+
+    public function  createBuilderInContext() returns Builder {
+        return new(self);
+    }
+
+    // Corresponds to LLVMContextDispose
+    public function dispose(){}
```

Comment:
Remove. This doesn't work with a GC language like Ballerina.

Reply:
Fixed